src/App.tsx

```diff
@@ -1,61 +1,131 @@
 import './App.scss';
+import { TodoList } from './components/TodoList';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import { FormEvent, useState } from 'react';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+type GetUserById = (todoId: number) => User | null;
+
+const getUserById: GetUserById = (todoId: number) => {
+  return usersFromServer.find(user => user.id === todoId) || null;
+};
```

Comment:
[CHECKLIST-5]: You should prepare all the data in the root component (App) and pass it to child components. Here you are fetching the user data directly in the function `getUserById`, which is not recommended. Instead, you should pass the user data from the server to the App component and then pass it down to the child components as needed.